### ISSUE TITLE
[Airbrake] [Staging] Couldn't find LoadProfile with 'id'=

### ISSUE THREAD

**Issue by dennisschoenmakers** [quintel/etmoses#1054]
**Airbrake error:** [#6358](https://quintel.airbrake.io/projects/111227/groups/1704252593693766358)
**Airbrake project:** ETMoses

**Error type:** `ActiveRecord::RecordNotFound`
**Error message:** `` Couldn't find LoadProfile with 'id'= ``
**Where:** `<no information>`
**Occurred at:** Jun 09, 2016 10:04:20 UTC
**First seen at:** Jun 09, 2016 10:04:20 UTC
**Occurrences:** 1 (1 since last deploy on Jun 09, 2016 09:31:48 UTC)

**URL:** `<no information>`
**File:** `[GEM_ROOT]/gems/activerecord-4.1.8/lib/active_record/relation/finder_methods.rb`

**Backtrace:**
```
[GEM_ROOT]/gems/activerecord-4.1.8/lib/active_record/relation/finder_methods.rb:320:in raise_record_not_found_exception!
[GEM_ROOT]/gems/activerecord-4.1.8/lib/active_record/relation/finder_methods.rb:429:in find_one
[GEM_ROOT]/gems/activerecord-4.1.8/lib/active_record/relation/finder_methods.rb:413:in find_with_ids
[GEM_ROOT]/gems/activerecord-4.1.8/lib/active_record/relation/finder_methods.rb:68:in find
```

**Comment by antw** [comment]
Caused by a must-run heat asset being set to the "---" profile.

**Comment by antw** [comment]
Tentatively closed by ef4820ce.
